tgui/packages/tgui/interfaces/MaterialStand.tsx

```diff
@@ -0,0 +1,120 @@
+import { createSearch, toTitleCase } from 'common/string';
+import { useBackend, useLocalState } from '../backend';
+import { Box, Button, Input, Stack, Flex, Section } from '../components';
+import { Window } from '../layouts';
+
+type Ores = {
+  id: string;
+  name: string;
+  amount: number;
+};
+
+type Ore_images = {
+  name: string;
+  icon: string;
+};
+
+type Data = {
+  ores: Ores[];
+  ore_images: Ore_images[];
+};
+
+export const MaterialStand = (props, context) => {
+  const { act, data } = useBackend<Data>(context);
+  const { ores } = data;
+  const [searchItem, setSearchItem] = useLocalState(context, 'searchItem', '');
+  const search = createSearch(searchItem, (ores) => ores.name);
+  const ores_filtered =
+    searchItem.length > 0 ? ores.filter((ore) => search(ore)) : ores;
+  return (
+    <Window title="MaterialStand" width={550} height={400}>
+      <Window.Content>
+        <Stack fill vertical>
+          <Section>
+            <Stack.Item>
+              <Input
+                autofocus
+                position="relative"
+                mt={0.5}
+                bottom="5%"
+                height="20px"
+                width="150px"
+                placeholder="Search Ore..."
+                value={searchItem}
+                onInput={(e, value) => {
+                  setSearchItem(value);
+                }}
+                fluid
+              />
+            </Stack.Item>
+          </Section>
+          <Section title="Stock" fill scrollable>
+            <Stack.Item>
+              <Flex wrap>
```

Comment:
Why the change to flex here? Is it just for wrap? I think this is on stack if so